src/resolver/__tests__/nodeModuleLookup.test.ts

```diff
@@ -114,20 +114,29 @@ describe('NodeModule lookup', () => {
 });
 
 describe('parseAllModulePaths', () => {
-  it('Should parse 1', () => {
-    const path = 'a/node_modules/@angular/core/node_modules/foo/node_modules/bar/far/woo/index.js';
+  it('Should skip node_modules directories', () => {
+    const path = '/a/node_modules/@angular/core/node_modules/foo/node_modules/bar/far/woo/index.js';
     const paths = parseAllModulePaths(path);
-    expect(paths[0]).toMatchFilePath(`a/node_modules$`);
-    expect(paths[1]).toMatchFilePath(`a/node_modules/@angular/core/node_modules$`);
-    expect(paths[2]).toMatchFilePath(`a/node_modules/@angular/core/node_modules/foo/node_modules$`);
-    expect(paths[3]).toMatchFilePath(`a/node_modules/@angular/core/node_modules/foo/node_modules/bar/node_modules$`);
-  });
-
-  it('Should give a primary path ', () => {
-    const path = 'some/user/file';
+    expect(paths).toStrictEqual([
+      '/node_modules',
+      '/a/node_modules',
+      '/a/node_modules/@angular/node_modules',
+      '/a/node_modules/@angular/core/node_modules',
```

Comment:
I think these tests will fail on windows, that's why there is a helper `toMatchFilePath `

Reply:
I don't know why I didn't think to check Windows.

Yes, but in fact, it's not just the test that is wrong.  The function itself will not work right on Windows because of the different behavior of absolute paths, which on windows contains a drive name.  I did just push a version that works on windows, but discovered it doesn't work in posix.  I'm going to redo the whole function and then I'll resolve this.